packages/wrangler/src/config/index.ts

```diff
@@ -156,6 +156,11 @@ export function readConfig(
 		if (!config.rules.some((rule) => rule.type === "PythonModule")) {
 			config.rules.push({ type: "PythonModule", globs: ["**/*.py"] });
 		}
+		if (!config.compatibility_flags.includes("python_workers")) {
```

Comment:
Nit: this `readConfig` function is growing and gaining more responsibilities. It's not the time or place for a big refactor, of course, but I'm wondering if we should extract the python validation code to a separate function to keep `readConfig` more readable as it grows.